storage/mysql/log_storage.go

```diff
@@ -101,6 +102,7 @@ func createMetrics(mf monitoring.MetricFactory) {
 	queuedCounter = mf.NewCounter("mysql_queued_leaves", "Number of leaves queued", logIDLabel)
 	queuedDupCounter = mf.NewCounter("mysql_queued_dup_leaves", "Number of duplicate leaves queued", logIDLabel)
 	dequeuedCounter = mf.NewCounter("mysql_dequeued_leaves", "Number of leaves dequeued", logIDLabel)
+	clearedAllStmtCounter = mf.NewCounter("mysql_cleared_all_prepared_statements", "Number of all prepared statements cleared")
```

Comment:
nit: -> "number of times the prepared-statement cache has been cleared"?

Reply:
fixed